electroncash_gui/qt/main_window.py

```diff
@@ -2667,6 +2682,26 @@ def resolve_cashacct(self, name):
         On failure throws up an error window and returns None.'''
         return cashacctqt.resolve_cashacct(self, name)
 
+    def resolve_lns(self, name):
+        ''' Throws up a WaitingDialog while it resolves an LNS Name.
+
+        Goes out to network, verifies all tx's.
+
+        Returns: a tuple of: (Info, Minimally_Encoded_Formatted_AccountName)
+
+        Argument `name` should be a Cash Account name string of the form:
```

Comment:
this description is referring to cash accounts

Reply:
resolved